python/huggingface_server.Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN cd huggingfaceserver && poetry install --no-interaction --no-cache
 
 RUN pip3 install vllm==${VLLM_VERSION}
 
-FROM nvidia/cuda:12.1.0-base-ubuntu22.04 as prod
+FROM nvidia/cuda:12.1.0-cudnn8-runtime-ubuntu22.04 as prod
```

Comment:
@yuzisun Do we need cuDNN ?

Reply:
is this needed for nccl installation ?

Reply:
`nvidia/cuda:12.1.0-runtime-ubuntu22.04` image comes with NCCL pre-installed. If we don't need cuDNN, We can use this image.